src/services/drpcEvents/index.ts

```diff
@@ -169,7 +169,7 @@ export default class DrpcEvents {
         params = submitQueryResponseRpcParams.parse(request.params)
       } catch (err) {
         this.logger.warn('Invalid parameters received for request %s: %o', request.id, request.params)
-        this.logger.debug('Parsing error o%', err)
+        this.logger.debug('Parsing error j%', err)
```

Comment:
why change this? 
I found this : 
-> Use %j if you only need a JSON-compatible output and the object is simple (without circular references).
-> Use %o when you need a detailed inspection of the object, especially in debugging scenarios.

Reply:
So I started with %o but then noticed that in pino-colada (if not mistaken) it was not rendering so since then instead of using %o I use %j also if not mistaken this could benefit searching log in tracer (to be implemented I believe) for example search by `req.id === 'some-id' && level  === '40'` across all services in this cluster, again depends on where we will forward those logs, but I think JSON is a safer option across the board

Reply:
ok